package_policies/M10.md

```diff
@@ -1,4 +1,5 @@
-**M10.** It is mandatory that each package have an xSDK-accessible **repository** (not necessarily
-publicly available), where the development version of the package is available. Support for taking **pull
-requests** is also recommended since this will greatly ease the effort among the development teams to
-improve interoperability and quickly provide bug fixes.
+**M10.** It is mandatory that each package have a **public repository**, for example at GitHub or
+Bitbucket, where the development version of the package is available. Support for taking **pull
+requests** is also recommended since this will ease greatly the effort for users to provide
```

Comment:
```suggestion
requests** is also recommended since this will  greatly ease the effort for users to provide
```
This was the prior text - and I think more appropriate format..

Alternative:

```suggestion
requests** is also recommended since this will ease the effort for users to provide
```